data/mods/More_Survival_Tools/items.json

```diff
@@ -113,480 +118,487 @@
+    "bashing": 1,
+    "material": [ "milk", "leather" ],
+    "symbol": ")",
+    "color": "white",
     "use_action": {
-      "type": "delayed_transform",
-      "transform_age": 28800, "//": "2 days",
-      "not_ready_msg": "The milk is still curdling.",
-      "msg": "The milk appears to have finished curdling, and is ready for further processing.  Checking on it has exposed the mixture to the atmosphere.",
-      "moves": 150,
       "target": "milk_curdled",
+      "msg": "The milk appears to have finished curdling, and is ready for further processing.  Checking on it has exposed the mixture to the atmosphere.",
+      "container": "waterskin",
       "target_charges": 6,
-      "container": "waterskin"
+      "moves": 150,
+      "type": "delayed_transform",
+      "transform_age": 28800,
+      "not_ready_msg": "The milk is still curdling.",
+      "//": "2 days"
     }
-  },{
-    "id": "milk_curdling2", "//": "Dummied out after mainlining. I'm always the last to hear about these things. -Chaosvolt",
+  },
+  {
+    "id": "milk_curdling2",
     "type": "TOOL",
-    "category" : "food",
-    "symbol": ")",
-    "color": "white",
+    "category": "food",
     "name": "batch of curdling milk",
     "name_plural": "batches of curdling milk",
+    "//": "price of container",
     "description": "A sealed waterskin filled with milk that is undergoing the process to become a crude form of cheese, having had vinegar and natural rennet added.",
-    "price": 4000, "//": "price of container",
-    "material": ["milk", "leather"],
     "weight": 3617,
     "volume": 14,
-    "bashing": 2,
+    "price": 4000,
     "to_hit": -1,
+    "bashing": 2,
+    "material": [ "milk", "leather" ],
+    "symbol": ")",
+    "color": "white",
     "use_action": {
-      "type": "delayed_transform",
-      "transform_age": 36000, "//": "2 and a half days",
-      "not_ready_msg": "The milk is still curdling.",
-      "msg": "The milk appears to have finished curdling, and is ready for further processing.  Checking on it has exposed the mixture to the atmosphere.",
-      "moves": 150,
       "target": "milk_curdled",
+      "msg": "The milk appears to have finished curdling, and is ready for further processing.  Checking on it has exposed the mixture to the atmosphere.",
+      "container": "waterskin2",
       "target_charges": 12,
-      "container": "waterskin2"
+      "moves": 150,
+      "type": "delayed_transform",
+      "transform_age": 36000,
+      "not_ready_msg": "The milk is still curdling.",
+      "//": "2 and a half days"
     }
-  },{
-    "id": "milk_curdling3", "//": "Dummied out after mainlining. I'm always the last to hear about these things. -Chaosvolt",
+  },
+  {
+    "id": "milk_curdling3",
     "type": "TOOL",
-    "category" : "food",
-    "symbol": ")",
-    "color": "white",
+    "category": "food",
     "name": "large batch of curdling milk",
     "name_plural": "large batches of curdling milk",
+    "//": "price of container",
```

Comment:
Can remove that, though this item has evidently been retained only to avoid load errors, it is no longer craftable according to comment higher up.